app/code/ElTrubetskaia/AskQuestion/view/frontend/web/js/ask_question.js

```diff
@@ -0,0 +1,99 @@
+define([
+    'jquery',
+    'Magento_Ui/js/modal/alert',
+    'mage/cookies',
+    'mage/translate',
+    'jquery/ui'
+], function ($, alert) {
+    'use strict';
+
+    $.widget('elTrubetskaia.askQuestion', {
+        options: {
+            cookieName: 'el_trubetskaia_ask_question_was_requested'
+        },
+
+        /** @inheritdoc */
+        _create: function () {
+            $(this.element).submit(this.submitForm.bind(this));
+        },
+
+        /**
+         * Validate request and submit the form if able
+         */
+        submitForm: function () {
+            if (!this.validateForm()) {
+                return;
+            }
+
+            this.ajaxSubmit();
+        },
+
+        /**
+         * Submit request via AJAX. Add form key to the post data.
+         */
+        ajaxSubmit: function () {
+            var formData = new FormData($(this.element).get(0));
+
+            formData.append('form_key', $.mage.cookies.get('form_key'));
+            formData.append('isAjax', 1);
+
+            $.ajax({
+                url: $(this.element).attr('action'),
+                data: formData,
+                processData: false,
+                contentType: false,
+                type: 'post',
+                dataType: 'json',
+                context: this,
+
+                /** @inheritdoc */
+                beforeSend: function () {
+                    $('body').trigger('processStart');
+                },
+
+                /** @inheritdoc */
+                success: function (response) {
+                    $('body').trigger('processStop');
+                    alert({
+                        title: $.mage.__(response.status),
+                        content: $.mage.__(response.message)
+                    });
+
+                    if (response.status === 'Success') {
+                        var  expired = new Date();
+                        expired.setMinutes(expired.getMinutes() + 2);
```

Comment:
1) Cookie is set here, but not validated. Request is sent in any case, even when the cookie is present.
2) Expiration date is not correct. See https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Set-Cookie
Expires= date Optional
Max-Age= number